dev/customer_testing/run_tests.dart

```diff
@@ -91,6 +105,21 @@ Future<bool> run(List<String> arguments) async {
     return help;
   }
 
+  if (files.length < shardIndex)
+    print('Warning: There are more shards than tests. Some shards will not run any tests.');
+
+  // Best attempt at evenly splitting tests among the shards
+  final List<File> shardedFiles = <File>[];
+  for (int i = shardIndex; i < files.length; i += numberShards) {
+    shardedFiles.add(files[i]);
+  }
+
+  if (verbose) {
+    print('Tests in this shard:');
+    for (final File file in shardedFiles)
+      print(file.path);
+  }
+
   if (verbose)
     print('Starting run_tests.dart...');
```

Comment:
this should be above all the prints you added, it's supposed to be the first thing in the output. :-)

Reply:
Thanks for pointing this out. Rearranged the prints to make more sense:

```
chillers@chillzone:~/Documents/flutter$ dart dev/customer_testing/run_tests.dart --verbose --shards 2 --shard-index 0 ~/flutter_tests/registry/*
Starting run_tests.dart...
8 files specified. 4 tests in shard #0.

Tests in this shard:
/usr/local/google/home/chillers/flutter_tests/registry/DanTup_tiler.test
/usr/local/google/home/chillers/flutter_tests/registry/flutter_gallery.test
/usr/local/google/home/chillers/flutter_tests/registry/flutter_svg.test
/usr/local/google/home/chillers/flutter_tests/registry/rainbowmonkey.test

Processing /usr/local/google/home/chillers/flutter_tests/registry/DanTup_tiler.test...
```